packages/bun-types/wasm.d.ts

```diff
@@ -0,0 +1,291 @@
+export {};
```

Comment:
Is all of this new? Or did we already have a version of this?

Reply:
This was just extracted from `globals.d.ts` into its own file, even tho this file still declares globals.
Perhaps naming it `global.wasm.d.ts` would be more descriptive.

Reply:
The current name is fine, just checking.